src/js/commands.js

```diff
@@ -128,6 +128,10 @@ class ClearCommand extends BaseCommand {
 
   run() {
     terminal.clearHistory();
+    return new BaseOutput(
```

Comment:
It is redundant to return an output which is anyways not gonna be used later. It is misdirecting any developer working on it, that this command is supposed to output **clear**